business/istio_validations.go

```diff
@@ -98,6 +100,8 @@ func (in *IstioValidationsService) GetValidations(namespace, service string) (mo
 
 	if service != "" {
 		objectCheckers = append(objectCheckers, in.getServiceCheckers(namespace, services, deployments, pods)...)
+	} else if workload != "" {
+		objectCheckers = append(objectCheckers, in.getWorkloadCheckers(namespace, rbacDetails.AuthorizationPolicies, workloadsPerNamespace[namespace])...)
 	}
 
 	// Get group validations for same kind istio objects
```

Comment:
There is one thing missing to be consistent with services though.
Right now, we are returning all the validation even we only need the ones related to the workload.
Do you mind add the else case in the following if? It will cast out validations not pointing to workloads.